cocos/base/ccUTF8.h

```diff
@@ -142,6 +142,43 @@ CC_DLL unsigned int getIndexOfLastNotChar16(const std::vector<char16_t>& str, ch
  */
 CC_DLL std::vector<char16_t> getChar16VectorFromUTF16String(const std::u16string& utf16);
 
+
+
+/**
+* Utf8 sequence
+* Store all utf8 chars as std::string
+* Build from std::string
+*/
+class CC_DLL StringUTF8
```

Comment:
One more question: Do we really need the StringUTF8 encapsulation? I think we have already provided many utf8, utf16 string conversion functions, why bother to provide another StringUTF8 class?
Thanks.